frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -86,45 +101,52 @@ Rider: ${ride.rider.firstName} ${ride.rider.lastName}`,
     );
   }, [drivers]);
 
-  const updateRides = (rideId: string, updatedDriver: Driver) => {
-    fetch(
-      `/api/rides/${rideId}`,
-      withDefaults({
-        method: 'PUT',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify({ driver: updatedDriver }),
-      }),
-    );
-  };
```

Comment:
we dont actually have an easy way to reassign drivers other than drag and drop, so we'll need to include it for now